subjects/cameltosnakecase/README.md

```diff
@@ -0,0 +1,56 @@
+## Camel-to-snake-case 
+
+### Instructions
```

Comment:
also the readme is still messy...
Use This:

```
### Instructions

Write a function that converts a string from `camelCase` to `snake_case`.

For this exercise you need to know that camelCase has two different writing alternatives that will be accepted:

- lowerCamelCase
- UpperCamelCase

Some rules for writing in camelCase:

- The word does not end on a capitalized letter (CamelCasE).
- No two capitalized letters shall follow directly each other (CamelCAse).
- Numbers or punctuation are not allowed in the word anywhere (camelCase1).

Here are some rules for you to follow:

- If the string is empty, return an empty string.
- If the string is not `camelCase`, return the string unchanged.
- If the string is `camelCase`, return the `snake_case` version of the string.
```

Reply:
Thank you for your suggestion. I change the description to this one.